jboss-eap-multinode-singlevm-rhel7/README.md

```diff
@@ -151,7 +148,7 @@ The deployment takes about 10 minutes to complete.
 
 ## Troubleshoot
 
-This section is to give you an idea on troubleshooting the issues that you might face when you deploy this template. If the parameter criteria are not fulfilled(like the Admin Password criteria) or if any parameter that are mandatory are not filled in the parameters section then the deployment will not start. Also make sure that you accept the terms and conditions mentioned before you click on Purchase. Once the deployment starts you can clearly see the resources getting deployed on the deployment page and if any deployment fails you can see which of the resources failed and check the deployment failure message for more details. If your deployment fails due to the script extension, you can see that the VM is already deployed so please login to the VM to check the logs for more troubleshooting. The logs are stored in the file jbosseap.install.log in path /var/lib/waagent/custom-script/download/0. The script can fail due to various reasons like incorrect RHSM Username/Password. This log file give you a clear message on why the script failed and exited. So you can use this log file to detect errors and troubleshoot them. Please run the following commands to check the logs once you login to the VM.
+This section provides information on troubleshooting some of the issues you might face when deploying this template. If the parameter criteria are not fulfilled (ex.- the Admin Password criteria) or if any mandatory parameters are not provided, the deployment will not start. Also the Terms & Conditions mentioned must be accepted before clicking on Purchase. Once the deployment starts the resources being deployed will be visible on the deployment page and in the case of any deployment failure a more detailed failure message is available.  If the deployment fails due to the script extension, you can see the VM is already deployed so login to the VM to check the logs for more troubleshooting detail. The logs are stored in the file jbosseap.install.log in path /var/lib/waagent/custom-script/download/0 or /var/lib/waagent/custon-script/download/1. The script can fail due to various reasons like incorrect RHSM Username/Password. This log file gives you a clear message on why the script failed and exited.  Use these logs to detect and correct errors.  Run the following commands to check the logs once you login to the VM.
```

Comment:
We have completely changed the structure of “Troubleshooting” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here.